clients/pkg/promtail/client/client.go

```diff
@@ -39,28 +39,35 @@ const (
 	ClientLabel     = "client"
 	TenantLabel     = "tenant"
 	DropReasonLabel = "reason"
+	MutationLabel   = "mutation"
 
 	DropReasonGeneric             = "ingester_error"
 	DropReasonRateLimited         = "rate_limited"
 	DropReasonStreamLimited       = "stream_limited"
 	DropReasongMaxLineSizeLimited = "max_line_size_limited"
+
+	MutationTruncated = "truncated"
```

Comment:
Let's reuse same reason `line_too_long` for both discarding and truncating long lines (won't be a backward incompatible as we haven't released this yet)

1. rename `DropReasongMaxLineSizeLimited` -> `ReasonLineTooLong` = "line_too_long"
2. remove `MutationTruncated`. We don't need this.

Now use `reason` label as `ReasonLineTooLong` in both truncated and discarded related metrics when max_line_size is exceeded.